data/categories.txt

```diff
@@ -7149,6 +7149,18 @@ el:Φάρος
 sk:Maják
 fa:فانوس دریایی
 
+man_made-survey_point
+en:4Survey Point|survey marker|survey benchmark|4geodetic mark|geodetic vertex|4triangulation station|4trigonometrical point|trig point|trig pillar|4trig station|trig beacon|trig
+ru:4Геодезический пункт|Геодезический знак|4тригонометрический пункт|пункт триангуляции|тригопункт|4триангулятор
+de:4Vermessungspunkt
+el:4Γεωδαιτικό σημείο
+et:4Geodeetiline märk
```

Comment:
Estonian language should be added in the code to work properly. I would recommend doing it separately in another PR.

Reply:
ah thank you! i've removed it now, hopefully there are no more issues :)